pkg/library/home/testdata/persistent-home/no-home-vm-when-disabled.yaml

```diff
@@ -6,6 +6,7 @@ input:
     workspace:
       persistUserHome:
         enabled: false
+        disableInitContainer: true
```

Comment:
Btw, was this added just for explicitness? I would imagine we don't need to set `disableInitContainer: false` if `persistUserHome.enabled` is set to `false`?

Reply:
That's right, I removed it actually since `disableInitContainer` isn't used at all if `persistUserHome.enabled` is false